tools/ci/tc/testdata/epochs_daily_push_event.json

```diff
@@ -0,0 +1,460 @@
+        "css/css-text/word-boundary/word-boundary-124.html",
+        "css/css-text/word-boundary/word-boundary-125.html",
+        "css/css-text/word-boundary/word-boundary-126.html",
+        "css/css-text/word-boundary/word-boundary-127.html",
+        "css/css-text/word-boundary/word-boundary-128.html",
+        "css/css-text/word-boundary/word-boundary-129.html"
+      ],
+      "removed": [
+
+      ],
+      "modified": [
+
+      ]
+    },
+    {
+      "id": "054edcc23aa1e0ebee50d7ddf1ce6115dd940ece",
+      "tree_id": "b58ca36bb7e928d440bed734e323580467dd32c7",
+      "distinct": false,
+      "message": "[css-text] Fix typo",
+      "timestamp": "2019-11-30T19:08:12+09:00",
+      "url": "https://github.com/web-platform-tests/wpt/commit/054edcc23aa1e0ebee50d7ddf1ce6115dd940ece",
+      "author": {
+        "name": "Florian Rivoal",
+        "email": "git@florian.rivoal.net",
+        "username": "frivoal"
+      },
+      "committer": {
+        "name": "Florian Rivoal",
+        "email": "git@florian.rivoal.net",
+        "username": "frivoal"
+      },
+      "added": [
+
+      ],
+      "removed": [
+
+      ],
+      "modified": [
+        "css/css-text/parsing/word-boundary-expansion-computed.html"
+      ]
+    },
+    {
+      "id": "d15d6d91834108a38070771025b548124d44026b",
+      "tree_id": "a1476d347b04acb59ed2562f0d4f845e8252e6d0",
+      "distinct": false,
+      "message": "[css-text add parsing tests for word-boundary-detection",
+      "timestamp": "2019-11-30T19:08:12+09:00",
+      "url": "https://github.com/web-platform-tests/wpt/commit/d15d6d91834108a38070771025b548124d44026b",
+      "author": {
+        "name": "Florian Rivoal",
+        "email": "git@florian.rivoal.net",
+        "username": "frivoal"
+      },
+      "committer": {
+        "name": "Florian Rivoal",
+        "email": "git@florian.rivoal.net",
+        "username": "frivoal"
+      },
+      "added": [
+        "css/css-text/parsing/word-boundary-detection-computed.html",
+        "css/css-text/parsing/word-boundary-detection-invalid.html",
+        "css/css-text/parsing/word-boundary-detection-valid.html"
+      ],
+      "removed": [
+
+      ],
+      "modified": [
+
+      ]
+    },
+    {
+      "id": "5df56b25e1cb81f81fe16c88be839f9fd538b41e",
+      "tree_id": "18da87e7701bd6218f4437b4d2d49cefe1f56af9",
+      "distinct": false,
+      "message": "Delete invalid test. (#20547)\n\nIt seems it expects `transform: rotate(1deg, 20px, 20px)` to somehow be valid.\r\n\r\nIt was introduced in cdc3032f56c86cc68121e54e169485441d9cdb1a, pointing to https://www.w3.org/TR/css-transforms-1/#svg-transform-functions, which doesn't say anything like that.\r\n\r\nDoesn't pass in any browser.",
+      "timestamp": "2019-11-30T13:34:24-08:00",
+      "url": "https://github.com/web-platform-tests/wpt/commit/5df56b25e1cb81f81fe16c88be839f9fd538b41e",
+      "author": {
+        "name": "Emilio Cobos Álvarez",
```

Comment:
Looks like Windows Python3 is choking on this, as it is assuming cp1252 on Windows. I think the 'open' call in `test_schedule_tasks` in `tools/ci/tc/tests/test_valid.py` probably needs to define an encoding (though @gsnedders knows a lot more about Python3 than me)